### ISSUE TITLE
Deep Learning Detection of Active Pulmonary Tuberculosis at Chest Radiography Matched the Clinical Performance of Radiologists

### ISSUE THREAD

**Issue by Soyeonnniiii** [iRAIL-SNU/irail-paper-review#29]
## Motivation

- 전 세계적으로 4명 중 1명이 결핵균에 감염되고, 이 중 5~10%가 활동성 결핵으로 나타남. 활동성 결핵의 90%가 의료 서비스에 접근할 자원이 부족한 30개의 "high-burden"국가에서 발생한다. 
-  2021년에 발행된 WHO 가이드라인에서 3개의 CAD system의 성능을 평가했고 진단 정확도나 성능이 human reader들과 비슷하다고 발표했다. 
-  본 논문에서는 흉부 X선 영상에서 활동성 결핵의 특징을 분석하는 DLS(deep learning system)을 개발했다. 데이터셋은 지역, 인종 또는 민족, 그 지역의 유병률을 반영한 다양한 국가의 통합 data set을 사용했다.

### Dataset
![image](https://user-images.githubusercontent.com/104473412/227436238-da4ed9ae-e30c-46c6-a176-e94eed78d827.png)
- Training : 160,187 images from Azerbaijan, Belarus, Georgia, Moldova, Romania, India and South Africa
- Validation(tuning) : 3,258 images from China, India and Zambia
- Test : 2,309 images from China, India, the United States, Zambia, South Africa(2,309)
- Pre-training : 550,297 images(10,310 images는 training과 validation set과 겹치나 test와는 겹치지 않음)
- Lung-crop model training and validating :  138 images from the U.S(lung segmentation mask labeled)

### DLS Description
 
![image](https://user-images.githubusercontent.com/104473412/227413110-7900a305-bf1f-47e0-8439-ee533f9a0b57.png)

이 모델은 CXR에서 활동성 결핵을 찾아내는 모델이며 3가지 모듈로 구성되어 있음
 - **a lung cropping model** : Faster-RCNN with a ResNet-101 feature extractor로 폐를 bounding box로 표시하고 bounding box를 사용해 그 부분만 잘라냄
- **a detection model** : Single Shot MultiBox Detector를 사용해 TB와 관련된 feature가 있는 부분을 bounding box로 표시(nodules, airspace opacities with cavitation, airspace opacities without cavitation, pleural effusion, granulomas, and fibroproductive lung opacities)

- **classification model** : 사전학습 된 EfficientNet-B7를 사용했고, lung cropping과 detection 모델의 output을 활용해 TB와 non-TB abnormality의 가능성을 예측

### Data set Characteristics
- Four country test set(China, India, U.S., Zambia) : 1,236(212TB confirmed culture or NAAT)
- One-country test set(South Africa) : 1,073(460TB) 
### DLS performance
![image](https://user-images.githubusercontent.com/104473412/227436485-e4e5c9ba-255c-46e2-8ced-9655bdf8060d.png)
- DLS와 radiologists에게 동일한 이미지를 보여줬음
- Validation threshold (0.45)로 고정했을 때 AUROC 0.89, DLS와 India-based radiologists의 sensitivity는 88.2% vs 74.6%와 specificity는 78.5% vs 84%로 sensitivity는 DLS가 더 높고 specificity는 noninferior함
### Comparison of India-based and U.S. based Radiologist
![image](https://user-images.githubusercontent.com/104473412/227437158-d7a051c8-acdb-4625-8934-5b20c8facdc2.png)
- 인도의radiologist 중 한 명은 양성 판독률이 현저하게 낮아 다른 india-based radiologist의 performance가 과소평가 되는 것을 방지하기 위해 제외
- 두 그룹 모두 DLS의 ROC곡선 보다 약간 아래에 있고, India-based가 specificity가 더 높았으며(84% vs 74%), sensitivity는 낮았음(74.6% vs 77.9%)
  - 전염병 케이스를 놓치는 위험을 줄이기 위해 양성의 가능성을 두고 테스트를 하는 진료 환경 때문 
- 수련 받은 지역에 관계없이 광범위한 sensitivity와 specificity를 보였음.
### Per-Data Set Analysis
![image](https://user-images.githubusercontent.com/104473412/227437358-90b80b84-1687-43a1-8204-1e28bf08906a.png)
- additional test data set 사용 (South Africa, 대부분 남성이며 광부) 
- 한 명의 radiologist만 ROC곡선 위에 있고 나머지는 다 아래에 있음
- 사전에 지정한 threshold(0.45, validation set)에서 DLS와 두 그룹의 radiologist와 비교했을 때 noninferior함
- threshold(0.685, South Africa validation data sets에서 높은 specificity를 가졌을 때)에서, India-based radiologist와 noninferior했으나 U.S-based radiologits보다 sensitivity는 낮지만 specificity는 더 높았음.
 ### Inter-Data Set Comparisons
![image](https://user-images.githubusercontent.com/104473412/227437847-6cf982c9-3955-4798-8aad-e6a05ec62ccb.png)
-  잠비아를 제외한 나머지 국가에서는 TB+와 TB-의 DLS prediction score에 대한 distribution이 유사했으나 잠비아에서는 TB-이미지가 전반적으로 모든 범위의 DLS prediction score에 걸쳐 있음. 
  - 잠비아 데이터셋은 사전에 다른 CAD screening으로  영상의학적으로 깨끗한 영상을 제거했고, TB는 음성이지만 다른 비정상 소견을 가지고 있는 영상이 많아 분류가 어려웠음 
 ### Visualizing Challenging Images
**1. Correct**
![image](https://user-images.githubusercontent.com/104473412/227438223-7d505f1f-a639-4372-b477-59e125965995.png)
- DLS value가 1또는 0에 가깝게 confident하게 예측한 것은 영상의학과 의사들의 해석과 높은 수준으로 일치했다.
- 반면에 DLS value가 0.45에 가까운 challenging 영상은 영상의학과 의사들의 performance도 감소했다. 
 **2. Incorrect**
![image](https://user-images.githubusercontent.com/104473412/227438643-6c9583f0-48ec-44f5-bb80-03028f475e15.png)
- DLS에서 예측이 틀린 TB+이미지(A)에서 영상의학과 의사들은 lymphadenopathy(TB-), military infection(TB+)으로, TB-이미지(B)에서는 bronchiolitis(TB+) and left lower lung infection(TB-)로 판독
### Matched performance to Radiologists and WHO Targets
-  Radiologist의 평균 specificity와 sensitivity에 따른 DLS의 sensitivity와 specificity를 비교했을 때 DLS의 성능이 noninferior했음
![image](https://user-images.githubusercontent.com/104473412/227438896-39455e8d-9b89-44fc-b32a-2cf6d4baf30b.png)
- WHO target sensitivityd인 90%로 고정했을 때 specificity는 76.6, specificity를 70%로 고정했을 때 specificity는 93.4로 잠비아와 남아프리카 data set을 제외하고 충족
### Subgroups according to HIV Status
![image](https://user-images.githubusercontent.com/104473412/227440014-94270897-3ba7-4f7e-ad4c-7d036e6673c9.png)
- HIV 환자가 있는 Zambia test dataset에서 평가를 했음. HIV(-)인 경우 DLS의 AUC는 0.92, HIV(+)인 경우 AUC는 0.8이었고 radiologist와 비교했을 때 sensitivity와 specificity가 약간 낮았다.
### Detecting Non-TB Pulmonary Findings
![image](https://user-images.githubusercontent.com/104473412/227440432-99a1ae9b-acad-430a-8912-3ef6fdfba03a.png)
- India와 Zambia test  dataset은 사전에 India-based radiologist가 라벨링을 했고, DLS에서 abnormality detector를 평가
- 양성 이미지는 TB-positive 또는 비정상 소견이 있다고 정의했고, 얼마나 많은 영상의학과 전문의들이 비정상 소견이라 판단했는지에 따라 다르지만 DLS의 AUC는 0.93~0.97이었다. 
  - 영상에대해 비정상이라고 1명이 한 경우 AUC 0.93, 2명이 한경우 0.97, 3명이 한 경우 0.97 
### Stratification according to TB-associated Radiologic Findings
![image](https://user-images.githubusercontent.com/104473412/227440782-600213f2-708b-4c3b-8ad4-929bcb19eb32.png)
- DLS는 TB와 관련된 영상의학적 abnormality에 높은 sensitivity를 가지고 있고 두 그룹의 radiologist 보다 높았음
- 10가지의 특이적인 비정상소견이 없을 경우에는 DLS와 radiologist 모두 sensitivity가 낮았음
- TB(+) 이미지 구별에 있어서 DLS가 비정상 소견에 의존하는 경향이 있음

## Insight
- TB high-burden국가의 다양한 인구군을 대상으로 연구를 함
- 다양한 방법으로  모델에 대한 성능을 평가함
- 의료서비스 자원이 부족한 지역에서 활용할 수 있다는 가능성 제시
- 결핵 유병률이 높은 국가들의 데이터를 활용했는데, real-world의 낮은 결핵 유병률 국가의 데이터에서도 좋은 성능을 보일까...?

### Paper URL

https://pubs.rsna.org/doi/full/10.1148/radiol.212213
